SpatialGDK/Documentation/content/toolbars.md

```diff
@@ -95,15 +95,33 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS GDK fo
 
  ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-settings.png)
 
-##### Configuration
+##### General
 
 | Setting | Description |
 | --- | --- |
 | SpatialOS directory | If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`. |
+
+##### Play In Editor Settings
+
+| Setting | Description |
+| --- | --- |
+| Delete Dynamically Spawned Entites | If enabled, deletes any dynamically spawned entities when taking down the running PIE session |
+
+##### Launch
+
+| Setting | Description |
+| --- | --- |
+| Generate default launch config | If checked, a launch configuration will be generated by default when launching spatial through the toolbar. |
 | Launch configuration | The [launch configuration file]({{urlRoot}}/content/glossary#launch-configuration-file) to use when running `spatial local launch` using the **Start** button. |
 | Stop on exit |  If enabled, shuts down running deployments when you close the Unreal Editor. |
+
+##### Snapshots
+
+| Setting | Description |
+| --- | --- |
 | Snapshot path | Use this to specify the filepath of your [snapshot]({{urlRoot}}/content/glossary#snapshot). If you leave this empty, it defaults to `<GameRoot>/../spatial/snapshots`. |
 | Snapshot file name |  Name of your snapshot file. |
+| Generate placeholder entities in snapshot | If checked, placeholder entities will be added to the snapshot on generation |
```

Comment:
```suggestion
| Generate placeholder entities in snapshot | If checked, the GDK adds placeholder entities to the snapshot when it is generated|
```

Reply:
Do we need to update https://docs.improbable.io/unreal/alpha/content/generating-a-snapshot#placeholder-entities if this is now an option?